android/src/test/resources/allure.properties

```diff
@@ -0,0 +1 @@
+allure.results.directory=target/allure-results
```

Comment:
Allure is like open source competitor to us right? So do we want to have allure integration in our repos?